### DISCUSSION TITLE
使用locale emulator后有办法缩放吗？

### DISCUSSION THREAD

**Discussion by sora987** [Blinue/Magpie#428] [Q&A] [answered]
求问使用[Locale Emulator](https://github.com/xupefei/Locale-Emulator)后，有办法缩放吗？Alt+F11似乎失效……

感谢！

**Comment by Blinue** [comment, marked as answer]
试试以管理员身份运行 Magpie 或使用“5秒后缩放”

**Reply by sora987** [reply]
感谢大佬，确实是管理员问题
